parquet-scrooge/src/test/java/parquet/scrooge/ScroogeStructConverterTest.java

```diff
@@ -18,106 +18,147 @@
  */
 package parquet.scrooge;
 
+import org.apache.thrift.TBase;
 import org.junit.Test;
-
-import parquet.scrooge.test.AddressWithStreetWithDefaultRequirement;
-import parquet.scrooge.test.TestFieldOfEnum;
-import parquet.scrooge.test.TestListPrimitive;
-import parquet.scrooge.test.TestMapComplex;
-import parquet.scrooge.test.TestMapPrimitiveKey;
-import parquet.scrooge.test.TestMapPrimitiveValue;
-import parquet.scrooge.test.TestOptionalMap;
-import parquet.scrooge.test.TestPersonWithAllInformation;
-import parquet.scrooge.test.TestSetPrimitive;
-import parquet.scrooge.test.TestUnion;
-import parquet.scrooge.test.StringAndBinary;
+import parquet.schema.MessageType;
+import parquet.scrooge.test.*;
 import parquet.thrift.ThriftSchemaConverter;
 import parquet.thrift.struct.ThriftType;
+
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
 
 /**
  * Test convert scrooge schema to Parquet Schema
  */
 public class ScroogeStructConverterTest {
+
+  /**
+   * Convert ThriftStructs from a thrift class and a scrooge class, assert
+   * they are the same
+   * @param scroogeClass
+   */
+  private void shouldConvertConsistentlyWithThriftStructConverter(Class scroogeClass) {
+    try {
+      Class<? extends TBase<?, ?>> thriftClass = (Class<? extends TBase<?, ?>>)Class.forName(scroogeClass.getName().replaceFirst("parquet.scrooge.test", "parquet.thrift.test"));
+      ThriftType.StructType structFromThriftSchemaConverter = new ThriftSchemaConverter().toStructType(thriftClass);
+      ThriftType.StructType structFromScroogeSchemaConverter = new ScroogeStructConverter().convert(scroogeClass);
+
+      assertEquals(structFromThriftSchemaConverter, structFromScroogeSchemaConverter);
+      assertEquals(toParquetSchema(structFromThriftSchemaConverter), toParquetSchema(structFromScroogeSchemaConverter));
+    } catch (ClassNotFoundException e) {
+      fail("ClassNotFoundException: " + e.getMessage());
+    }
```

Comment:
why not just let the exception propagate up? At least we would have the correct stacktrace